doxygen/01_Installation.dox

```diff
@@ -105,7 +105,48 @@ window used. One can open an instance of CMD.EXE with the development
 environment already set up by navigating to Start - All Programs - 
 Microsoft Visual Studio - Visual Studio Tools - x64 Native Tools Command Prompt. You may also wish to
 create a shortcut for this tool on the desktop, for convenience.
-     
+
+\section pygenn Installing PyGeNN
+Building PyGeNN is a little involved so we recommend installing a prebuilt binary wheel from our Releases page or direct from our <a href="https://gen-ci.inf.sussex.ac.uk/job/GeNN/job/genn/job/master/">build server</a>.
+
+\subsection pygenn_wheel Installing PyGeNN from binary wheels
+ - Select a suitable wheel from the Releases page (all wheels are build for CUDA 10 except for on Mac OS which is built for CUDA 9). For example, if you have a Linux system with Python 3.7 you would pick ``pygenn-0.2.1-cp37-cp37m-linux_x86_64.whl``. If you do not have CUDA installed, ignore the CUDA version and pick the wheel that otherwise matches your platform.
+ - Install the wheel using pip:
+ \code
+ pip install pygenn-0.2.1-cp37-cp37m-linux_x86_64.whl
+ \endcode
+ 
+\subsection pygenn_source_nix Installing PyGeNN from source on Linux or Mac OSX
+ - Navigate to the GeNN directory and build a dynamic library version of GeNN, directly into the PyGeNN directory with:
+ \code
```

Comment:
tried this on Mac OS 11.0.3 ... a couple of observations:
- it needs swig installed (and swig-python if using swig from macports)
- it looks as if it works but when I tried one of Manvi's examples I get "Segmentation fault: 11" immediately ... not sure what is broken (this is probably not the place to discuss)